apps/condo/domains/acquiring/routes/paymentLinkRouter.js

```diff
@@ -187,22 +210,61 @@ class PaymentLinkRouter {
     }
 
     isVirtualReceipt (req) {
-        const { billingReceiptId } = this.extractRegularReceiptParams(req)
+        const { billingReceiptId, invoicesIds } = this.extractRegularReceiptParams(req)
```

Comment:
why do not use both extractregularReceiptParams and extractInvoicesParams togeher?

Reply:
Refactored